extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/ranger/RangerSparkExtensionSuite.scala

```diff
@@ -380,4 +384,55 @@ class InMemoryCatalogRangerSparkExtensionSuite extends RangerSparkExtensionSuite
 
 class HiveCatalogRangerSparkExtensionSuite extends RangerSparkExtensionSuite {
   override protected val catalogImpl: String = "hive"
+
+  test("table stats must be specified") {
+    val table = "hive_src"
+    try {
+      doAs("admin", sql(s"CREATE TABLE IF NOT EXISTS $table (id int)"))
+      doAs(
+        "admin", {
+          val hiveTableRelation = sql(s"SELECT * FROM $table")
+            .queryExecution.optimizedPlan.collectLeaves().head.asInstanceOf[HiveTableRelation]
+          assert(getFieldVal[Option[Statistics]](hiveTableRelation, "tableStats").nonEmpty)
+        })
+    } finally {
+      doAs("admin", sql(s"DROP TABLE IF EXISTS $table"))
+    }
+  }
+
+  test("HiveTableRelation should be converted to LogicalRelation") {
+    val table = "hive_src"
+    try {
+      doAs("admin", sql(s"CREATE TABLE IF NOT EXISTS $table (id int) STORED AS PARQUET"))
+      doAs(
+        "admin", {
+          val relation = sql(s"SELECT * FROM $table")
+            .queryExecution.optimizedPlan.collectLeaves().head
+          assert(relation.isInstanceOf[LogicalRelation])
+        })
+    } finally {
+      doAs("admin", sql(s"DROP TABLE IF EXISTS $table"))
+    }
+  }
+
+  test("Pass through JoinSelection") {
+    val db = "test"
+    val table1 = "table1"
+    val table2 = "table2"
+    try {
```

Comment:
we can doAs { "admin", { 
 try {
   sql 1
   sql 2
   sql 3
 } finally...
 }